tier4/{{cookiecutter.project_slug}}/.github/codejson/hooks/post_gen_project.py

```diff
@@ -37,6 +37,10 @@ def get_scc_labor_hours():
 
         #Otherwise just use previous value as a default value.
         return None
+    
+def prompt_exemption_text():
+    print("You have selected an exemption for your Usage Type.")
```

Comment:
Could we add the exemption value the user selected to this print statement here, just so they know what we are referring to